components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -529,11 +537,19 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
             .unwrap();
         trace.try_persist = false;
         trace.persisted_applied = trace.admin.flushed;
+
+        self.set_flushed_epoch(epoch);
```

Comment:
The region has not been persisted yet, is it okay to update flushed epoch?

Reply:
ditto